src/main/resources/mcmod.info

```diff
@@ -11,6 +11,6 @@
       "credits": "",
       "logoFile": "assets/refinedstorage/textures/logo.png",
       "screenshots": [],
-      "dependencies": []
+      "dependencies": ["inventorytweaks"]
```

Comment:
You are adding support, a dependency would always need to be present for RS to function. Which isn't the case here.